db/migrate/20170427130832_rename_section_edition_section_id_to_section_uuid.rb

```diff
@@ -0,0 +1,13 @@
+class RenameSectionEditionSectionIdToSectionUuid < Mongoid::Migration
+  def self.up
+    SectionEdition.all.each do |section_edition|
+      section_edition.rename(:section_id, :section_uuid)
+    end
+  end
+
+  def self.down
+    SectionEdition.all.each do |section_edition|
+      section_edition.rename(:section_uuid, :section_id)
+    end
+  end
+end
```

Comment:
I ran it locally using recent data from production and it ran in a matter of seconds. I'm hoping that's going to be representative of how it'll run in production too!

I considered using the bulk update command but decided against it when I saw that this version seems to run quickly enough.